data/blog/2024/2024-09-26-using-laminas-continuous-delivery-and-deployment.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 2024-09-26-using-laminas-continuous-delivery-and-deployment.md
+author: julian
+title: 'Using Laminas Continuous Delivery and Deployment'
+draft: false
+public: true
+created: '2024-09-26T11:00:00-01:00'
+updated: '2024-09-26T11:00:00-01:00'
+tags:
+  - continuous integration
+---
+
+We detailed the meaning and some inner workings of Continuous Integration (**CI**) in a [previous article](https://getlaminas.org/blog/2024-08-05-using-laminas-continuous-integration.md.html), so here we will focus on what happens after CI completes successfully.
```

Comment:
Something went wrong here: `.md.html` ?!

Reply:
But what is the reason for this problem?

> and now is too late to fix the old issue, since the link was propagated as is now…

Is it not possible to set a redirect somewhere? We also need this, for example, if we want to restructure the content.

Reply:
> But what is the reason for this problem?

In the previous article , in the *front matter* section, it was 
"id: 2024-08-05-using-laminas-continuous-integration.md" 
instead of 
"id: 2024-08-05-using-laminas-continuous-integration"

> Is it not possible to set a redirect somewhere? We also need this, for example, if we want to restructure the content.

Is possible to set a redirect with 301 in htaccess file. But i prefer to do it in a separate PR, right after this PR will be merged.

Reply:
I created an issue for that , https://github.com/laminas/getlaminas.org/issues/211